tests/foreman/ui/test_contenthost.py

```diff
@@ -226,6 +201,21 @@ def test_positive_end_to_end(session, default_location, repos_collection, vm):
 
 @pytest.mark.upgrade
 @pytest.mark.tier3
+@pytest.mark.parametrize(
+    'module_repos_collection_with_manifest',
+    [
+        {
+            'distro': DISTRO_RHEL7,
+            'RHELAnsibleEngineRepository': {'cdn': True},
+            'SatelliteToolsRepository': {},
+            'YumRepository': [
+                {'url': settings.repos.yum_1.url},
+                {'url': settings.repos.yum_6.url},
+            ],
+        },
+    ],
+    indirect=True,
+)
```

Comment:
above block has been repeated more than at-least 10 times times. Can this be moved to common place at module level ?

Reply:
Since the tests are using fixtures like in this case `test_positive_end_to_end_bulk_update` using `vm` fixture and the `vm` fixture is using `module_repos_collection_with_manifest` fixture. The `module_repos_collection_with_manifest` fixture need these parameters on the test to resolve `requests.param` in its definition. and Hence these parameters are repeating every-time the repos_collection fixtures are being used.

Reply:
I think @omkarkhatavkar has a point here about the repetition, and this would be a good time to use a `TestClass` wrapper for all of the tests needing this indirect parametrization, so that it can be applied once to the class.

Non-blocking for the PR, but certainly something the component owner can improve on moving forward.